src/libNode/Node.cpp

```diff
@@ -154,8 +154,10 @@ bool Node::DownloadPersistenceFromS3() {
   string output;
   // TBD - find better way to capture the exit status of command
   string excludembtxns = LOOKUP_NODE_MODE ? "false" : "true";
-  SysCommand::ExecuteCmdWithOutput(
-      "./downloadIncrDB.py " + STORAGE_PATH + "/ " + excludembtxns, output);
+  SysCommand::ExecuteCmdWithOutput(boost::filesystem::current_path().string() +
```

Comment:
boost::filesystem::current_path().string() should included zilliqa already, we should log the command the software executed

Reply:
Like @ShengguangXiao suggested, can we log and verify if the download script works as expected?